ltp-production.yaml

```diff
@@ -1563,3 +1566,39 @@ projects:
     url: https://bugs.linaro.org/show_bug.cgi?id=5766
     active: true
     intermittent: true
+  - environments: *environments_32bit
+    notes: >
+      LTP: futex_wake04: TBROK: Failed to open FILE '/proc/11583/task/11585/stat' for reading: ENOENT (2)
+    projects: *projects_all
+    test_names:
+    - ltp-syscalls-tests/futex_wake04
+    url: https://bugs.linaro.org/show_bug.cgi?id=5768
+    active: true
+    intermittent: true
+  - environments: *environments_arm64_arm32
+    notes: >
+      LTP: RTNETLINK answers: Operation not supported
+      netns_breakns 1 TBROK: unable to create veth pair devices
+      Cannot find device \"veth0\"
+    projects:
+    - lkft/linux-stable-rc-linux-4.4.y
+    test_names:
+    - ltp-containers-tests/netns_breakns_ns_exec_ipv4_netlink
```

Comment:
@nareshkamboju Just FYI, you can use the `*` pattern in other places as well. These could be renamed to:

```yaml
test_names:
- ltp-containers-tests/netns_*_netlink
- ltp-containers-tests/netns_*_ioctl
```

But I don't know if it's what you want

Reply:
@chaws I have made changes as per your comments.